OOP/hw1.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class Shape
+{
+    public $width;
+    public $height;
+    
+
+    function __construct($width, $height) {
+        $this->width = $width;
+        $this->height = $height;
+    }
+}
+
+class triangle extends shape
```

Comment:
1. Classname should be `PascalCase` (Triangle)
2. Extends from `Shape` not `shape`